{{ cookiecutter.project_slug }}/docs/pre_commit_hook.md

```diff
@@ -2,11 +2,18 @@
 
 To run additional checks before making commit we use [Pre-commit](https://pre-commit.com/) hooks.
 
-`pre-commit` package is installed with backend requirements, so there is no need of [manual installation](https://pre-commit.com/#install).
-But you need to generate the actual git pre-commit hook. It should be done only once:
+
+Since the `api` is running inside docker container we don't have a pre-commit as a dependency there, so you need to 
+install `pre-commit` package [manually](https://pre-commit.com/#install).
```

Comment:
I think this make more sense in this way. Not sure about "locally" but I want to emphasize that pre-commit needs to be installed system wise 🤔 
```suggestion
Since the `api` is running inside docker container and we don't have a pre-commit as a dependency there, you need to 
install `pre-commit` package [locally](https://pre-commit.com/#install).
```